README.md

```diff
@@ -97,7 +97,7 @@ This plugin intends to support linting of ES2015+ (ES6+) import/export syntax, a
 
 <!-- end auto-generated rules list -->
 
-## `eslint-plugin-import` for enterprise
```

Comment:
Please help to remove this section, this package is not part of Tidelift subscription.

Reply:
removed the entire chapter